matrix/common/request/request_tracker.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, request_hash: str):
         Logging.set_correlation_id(logger, request_hash)
 
         self.request_hash = request_hash
+        self._num_bundles = None
```

Comment:
Why instantiate this instead of just using the property?

Reply:
This variable caches the value in the table